### PR TITLE
Added dashboard for sig-storage.

### PR DESCRIPTION
**Pull request by brahmaroutu**
Attempting to add dashboard for sig-storage similar to sig-cli.

### CODE REVIEW

File: testgrid/config/config.yaml [hunk 1]
```diff
@@ -1601,6 +1601,71 @@ dashboards:
...
+    test_group_name: ci-kubernetes-e2e-gce-serial
+    base_options: 'include-filter-by-regex=Volume'
+    description: 'storage gce serial e2e tests for master branch'
+  - name: gke-serial
+    test_group_name: ci-kubernetes-e2e-gke-serial
+    base_options: 'include-filter-by-regex=Volume'
+    description: 'storage gke serial e2e tests for master branch'
+  - name: unit-integration-cmd
+    test_group_name: ci-kubernetes-test-go
+    # test-cmd, test-integration do not produce xml data right now
```
Comment (fejta) [inline testgrid/config/config.yaml]: test-cmd is definitely irrelevant to volume, no need to mention it.
Does node have any integration tests? I doubt it, and suspect that can be removed as well.

File: testgrid/config/config.yaml [hunk 2]
```diff
@@ -1601,6 +1601,71 @@ dashboards:
     base_options: 'include-filter-by-regex=pkg/kubectl%7COverall'
     description: 'kubectl e2e tests in gke-master-1.6-gci-kubectl-skew-serial'
 
+- name: sig-storage
```
Comment (fejta) [inline testgrid/config/config.yaml]: How about also including the node_e2e tests?

**Comment by fejta** [comment]
/assign @dchen1107 
@k8s-bot ok to test

File: testgrid/config/config.yaml (hunk 2 above)
Comment (fejta) [inline testgrid/config/config.yaml]: Sorry, nevermind we don't want the node e2e tests in sig-storage

File: testgrid/config/config.yaml (hunk 1 above)
Comment (brahmaroutu) [inline testgrid/config/config.yaml]: I thought that this will show unit tests relevant to storage?

Comment (fejta) [inline testgrid/config/config.yaml]: Please don't just blindly copy stuff from sig-cli. Nothing about test-cmd has any relevance to storage.

I am not sure whether test-integration has any relevance

**Comment by fejta** [comment]
Thanks! Let's ship it. Hopefully the @kubernetes/sig-storage-misc will improve this over time.

**Pull request merged**